### PR TITLE
feat: Automatically get checkpoint on startup

### PR DESCRIPTION
**Pull request by zvolin**

### CODE REVIEW

File: src/ethereum_canister_backend/src/helios.rs [hunk 1]
```diff
@@ -68,3 +70,22 @@ where
...
+        serde_json::from_slice(&header_resp.body).wrap_err("Reading json response failed")?;
+    let checkpoint = header
+        .pointer("/data/root")
+        .and_then(Value::as_str)
+        .ok_or_else(|| {
+            eyre!(
+                "No root found in response: {}",
+                std::str::from_utf8(&header_resp.body).expect("Non utf-8 json")
+            )
+        })?;
```
Comment (oblique) [inline src/ethereum_canister_backend/src/helios.rs]:
```suggestion
    let body = str::from_utf8(&header_resp).wrap_err("Non utf-8 response")?;
    let header: Value =
        serde_json::from_str(body).wrap_err("Reading json response failed")?;
    let checkpoint = header
        .pointer("/data/root")
        .and_then(Value::as_str)
        .ok_or_else(|| eyre!("No root found in response: {body}")))?;
```

**Comment by oblique** [comment]
https://github.com/eigerco/helios/pull/7 is merged. You can update `rev` in Cargo.toml

**Comment by zvolin** [comment]
we still have paths in Cargo.toml, I'm changing it with the next tests PR

**Review by oblique — APPROVED** [review]
lgtm

**Pull request merged**